monai/metrics/active_learning_metrics.py

```diff
@@ -123,3 +158,48 @@ def compute_variance(
         elif scalar_reduction == "sum":
             var_sum = torch.sum(variance)
             return var_sum
+
+
+def label_quality_score(
+    y_pred: torch.Tensor, y: torch.Tensor, include_background: bool = True, scalar_reduction: str = "mean"
+):
+    """
+    The assumption is that the DL model makes better predictions than the provided label quality, hence the difference
+    can be treated as a label quality score
+
+    Args:
+        y_pred: [B, C, H, W, D] or [B, C, H, W] or [B, C, H] where B is Batch-size, C is channels and H, W, D stand for
+        Height, Width & Depth
+        include_background: Whether to include the background of the spatial image or channel 0 of the 1-D vector
+        scalar_reduction: reduction type of the metric, either 'sum' or 'mean' can be used to retrive a single scalar
+        value, if set to 'none' a spatial map will be returned
```

Comment:
could you please fix the indentation of the docstrings? you can look at the other classes as examples. https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#building-the-documentation

Reply:
I am not sure which indentations to fix. I was following the docstrings of this particular [meandice](
https://github.com/Project-MONAI/MONAI/blob/dev/monai/metrics/meandice.py) for reference.

Will fix once I understand more.

Reply:
Sure, I put more details here so that it's useful as a reference in the future.

The docstring here will eventually be rendered on `docs.monai.io` page after merging the PR: https://docs.monai.io/en/stable/metrics.html. The process is automated and to locally generate the html pages for testing purposes, please follow this section https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#building-the-documentation.

After running the doc build commands, the generated html will be located at `/your_project_clone_folder/MONAI/docs/build/html/metrics.html` The current results by viewing the page from a web browser looks like the following:
![Screenshot 2022-09-09 at 07 26 37](https://user-images.githubusercontent.com/831580/189285682-c5121170-3606-4768-a5f4-ed926522262d.png)

As I annotated in the screenshot, there are some format issues, because of the indentation in the source code docstring not following the reStructuredText format:

![Screenshot 2022-09-09 at 07 36 36](https://user-images.githubusercontent.com/831580/189286878-bd6f1f41-ca51-464f-8cec-d689521e88e4.png)

For more information about the reStructured Text format: 

- https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

more details on the docstring style that we follow here: 

- https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

Please let me know if you have any further question, I'll update this comment to make it a self-contained documentation and put it on the project wiki.